build/Targets/Versions.props

```diff
@@ -0,0 +1,10 @@
+<Project>
+  <PropertyGroup>
+    <FakeSignVersion>0.9.2</FakeSignVersion>
+    <RoslynVersion>2.3.0-beta2-61719-05</RoslynVersion>
+    <RoslynAnalyzersVersion>1.3.0-beta1</RoslynAnalyzersVersion>
```

Comment:
@jinujoseph, what version is this supposed to be? We don't have a 1.3.0-beta1 version, there are a bunch of warnings during package restore about getting 2.0.0-beta1.